internal/nodes/avi_model_advl4_translator.go

```diff
@@ -447,7 +447,11 @@ func (o *AviObjectGraph) ConstructSharedVipSvcLBNode(sharedVipKey, namespace, ke
 		}
 
 		if i == 0 {
-			sharedPreferredVIP = svcObj.Spec.LoadBalancerIP
+			if svcObj.Spec.LoadBalancerIP != "" {
+				sharedPreferredVIP = svcObj.Spec.LoadBalancerIP
+			} else if svcObj.Spec.LoadBalancerIP == "" && svcObj.Annotations[lib.LoadBalancerIP] != "" {
```

Comment:
@saurabhvaidvmware: I think we can avoid first check `svcObj.Spec.LoadBalancerIP== ""`. This should be done everywhere.

Reply:
Dropped this condition check.